src/controllers/DrawingModeController.js

```diff
@@ -104,7 +113,18 @@ class DrawingModeController extends AnnotationModeController {
         /* eslint-enable require-jsdoc */
 
         this.drawingStartHandler = this.drawingStartHandler.bind(this);
-        this.pushElementHandler(this.annotatedElement, ['mousedown', 'touchstart'], this.drawingStartHandler);
+        this.pushElementHandler(
+            this.annotatedElement,
+            'click',
+            (event) => {
```

Comment:
Can you extract this function?

Reply:
Did it for all the drawing handlers while i was at it